actors/datacap/src/v13/state.rs

```diff
@@ -0,0 +1,37 @@
+use frc46_token::token;
+use fvm_ipld_blockstore::Blockstore;
+use fvm_ipld_encoding::tuple::*;
+use fvm_shared4::address::Address;
+use fvm_shared4::econ::TokenAmount;
+use fvm_shared4::error::ExitCode;
+use fvm_shared4::ActorID;
+
+use fil_actors_shared::v13::{ActorError, AsActorError};
+
+#[derive(Debug, Serialize_tuple, Deserialize_tuple)]
+pub struct State {
+    pub governor: Address,
+    pub token: token::state::TokenState,
```

Comment:
Why does it have the _official_ token whilst in the other datacap states we have a copy-pasted one? Can we always use the crate?